packages/shared/src/lib/featureManagement.ts

```diff
@@ -30,7 +30,7 @@ export class Features<T extends FeatureValue = string> {
     ],
   );
 
-  static readonly Search = new Features('search', SearchExperiment.Control);
+  static readonly Search = new Features('search', SearchExperiment.V1);
```

Comment:
Do write a TODO or something, just so we 100% don't forget this 😅

Reply:
Oh, right. We will have to keep this back to Control.